app/views/admin/navigation_links/_form.html.erb

```diff
@@ -9,19 +9,19 @@
   <div class="alert alert-info">A full (external) or relative (internal) URL for the link</div>
 </div>
 <div class="form-group" data-controller="svg-icon-upload">
-  <div nav-link-id="<%= form.object[:id] %>">
+  <div data-svg-icon-upload-target="navId" nav-link-id="<%= form.object[:id] %>">
     <%= form.label :icon %>
-    <div id="svg-icon-preview" class="<%= form.object[:id].nil? ? "" : "pb-3" %>" style="width: 24px;">
+    <div data-svg-icon-upload-target="svgIconPreview" class="<%= form.object[:id].nil? ? "" : "pb-3" %>" style="width: 48px;">
       <%= form.object[:icon]&.html_safe %>
     </div>
-    <%= form.hidden_field :icon, id: "svg-icon-content" %>
+    <%= form.hidden_field :icon, "data-svg-icon-upload-target": "svgIconContent" %>
 
     <input
-      type="file" accept="image/svg" class="form-control"
+      type="file" accept="image/svg, image/svg+xml" class="form-control"
```

Comment:
Not sure all browser support the `, `, according to the specs there should be no space between the types. Given that, `image/svg` is not needed as it doesn't exist :), SVG has only `image/svg+xml` as media type: https://www.w3.org/TR/SVG11/mimereg.html

Reply:
Oh, okay understood.